src/sidebar/components/ShareDialog/ExportAnnotations.tsx

```diff
@@ -29,6 +31,11 @@ function ExportAnnotations({ annotationsExporter }: ExportAnnotationsProps) {
   const draftCount = store.countDrafts();
 
   const inputRef = useRef<HTMLInputElement | null>(null);
+  const defaultFilename = useMemo(
+    () => suggestedFilename({ groupName: group?.name }),
+    [group],
+  );
+  const [filenameIsValid, setFilenameIsValid] = useState(true);
```

Comment:
The conventional approach here would be to store the input value as state and then derive `filenameIsValid` from that. The state would be initialized to `defaultFilename`.

Reply:
I also feel inclined to make the input a controlled one, as we need a piece of state anyway.